com.moddb.TotalChaosRetro.yaml

```
@@ -0,0 +1,82 @@
+app-id: com.moddb.TotalChaosRetro
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gzdoom.sh
+finish-args:
+  - --device=all
+  - --socket=wayland
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

since it's buggy with wayland

---

New GZDoom engine is not buggy as legacy one.